WinNSTabView.m

```diff
@@ -0,0 +1,228 @@
+//
+//  WinNSTabView.m
+//  WinUXTheme
+//
+//  Created by Marcian Lytwyn on 1/9/18.
+//  Copyright © 2018 Marcian Lytwyn. All rights reserved.
+//
+
+#import "WinUXTheme.h"
+
+@implementation WinUXTheme (NSTabView)
+
+static int _TabStateForThemeControlState(NSTabState state)
+{
+  switch (state)
+  {
+    case NSPressedTab:
+      return TIS_FOCUSED;
+    case NSSelectedTab:
+      return TIS_SELECTED;
+    case NSBackgroundTab:
+    default:
+      return TIS_NORMAL;
+  }
+}
+
+- (void) drawTabViewBezelRect: (NSRect)aRect
+                  tabViewType: (NSTabViewType)type
+                       inView: (NSView *)view
+{
+  if (!IsThemeActive())
```

Comment:
Could you please restructure this method? It has three time the same super call. By inverting all the if statements, plus adding a return in the middle, this could be simplified a lot